setup_README.md

```diff
@@ -0,0 +1,31 @@
+The following are the two setup steps that need to be performed once at the start of a project.  
```

Comment:
I think of each batch as possibly being a separate experiment, and these steps don't need to happen with each batch. So if you don't like "project" and I don't like "experiment", can we come up with another word?

Reply:
And/or we should put into the documentation how we define project/experiment/batch?

Reply:
Want to say "start of each batch of data collection" or "start of each project batch"? Let's keep **only once** though.

I agree the data pipeline welding "unit" will be experimental batch. An experiment may contain multiple batches, and a project may contain multiple experiments (in my view). 

Although we may eventually want to make the recipe focused at the "experiment" level (as defined above) since we are likely to want to develop batch effect correction methods. These tools should be part of the recipe IMO - this is a bit down the road though

Reply:
I just added a commit that defines our terms and so it should be consistent now.

I agree we want to eventually bring it to the experiment level ([related issue](https://github.com/broadinstitute/pooled-cell-painting-profiling-recipe/issues/20)).